config/test/com/microsoft/azure/iotsolutions/uiconfig/services/StorageTest.java

```diff
@@ -350,11 +352,93 @@ public void updateDeviceGroupAsyncTest() throws BaseException, ExecutionExceptio
                 Mockito.any(String.class),
                 Mockito.any(String.class))).
                 thenReturn(CompletableFuture.supplyAsync(() -> model));
-        storage = new Storage(mockClient, config);
         DeviceGroup result = storage.updateDeviceGroupAsync(groupId, group, etagOld).toCompletableFuture().get();
         assertEquals(result.getId(), groupId);
         assertEquals(result.getDisplayName(), displayName);
         assertEquals(result.getConditions(), conditions);
         assertEquals(result.getETag(), etagNew);
     }
+
+    @Test
+    @Category({UnitTest.class})
+    public void addPackageTest() throws BaseException, ExecutionException, InterruptedException {
```

Comment:
// Arrange ... in this file for new tests.

Reply:
Done